templates/students/update_profile.html

```diff
@@ -28,31 +28,24 @@
        opacity: 1;
 
     }
-    .caption {
-       background-color: rgba(128,128,128,0.5);
-       color: white;
-       font-size: 30px;
-       padding-top:30%;
-       border-radius: 6px;
-       height: 100%;
-       font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
-       font-weight: bolder;
-       z-index: 10;
-    }
-    .hoverText {
-       position: absolute;
-       top:0;
-       height: 150px;
-       transition: .5s ease;
-       opacity: 0;
-       width: 150px;
-       text-align: center;
-    }
-    #container {
-      width: 100px;
-      height: 100px;
-      position: relative;
-}
+    
+  #profile-image-popover{
+  position: relative;
+  }
+  #profile-image-popover:after{
+  position: absolute;
+  }
+  #profile-image-popover:hover svg{
+    opacity: 1;
+    color: white;
+  }
+  svg{
+    position:absolute;
+    top: 65px;
+    left: 65px;
```

Comment:
Use 
```
top: 50%;
left: 50%;
transform: translate(-50%, -50%);
```
instead to centre the SVG instead of hard-coding the px values.

Reply:
Done